### ISSUE TITLE
display the config for a package that does not have configuration, a bad error message is displayed

### ISSUE THREAD

**Issue by wdpoorte** [habitat-sh/habitat#1729]
This can be emulated with the following commands

```
[chef@ip-172-31-18-110 ~]$ sudo -E hab pkg install  core/logstash
» Installing core/logstash
→ Using core/logstash/5.1.1/20161221223434
★ Install of core/logstash/5.1.1/20161221223434 complete with 0 new packages installed.
[chef@ip-172-31-18-110 ~]$ sudo -E hab sup config  core/logstash
hab-sup(ER)[src/error.rs:372:8]: No such file or directory (os error 2)
```

Hello - thanks for reporting an issue with Habitat.

In order to help us troubleshoot the issue, please be sure to include the following information if applicable:

- [ ] The OS (including version) where you are running any of the Habitat commands.
Centos 7.2
- [ ] Debug/backtrace of the command you are trying to run. You can set the following environment variables before running the `hab` command to generate a trace:

```
RUST_LOG=debug RUST_BACKTRACE=1 hab sup config  core/logstash
DEBUG:habitat_common::ui: InputStream(stdin): { is_a_terminal(): true }
DEBUG:habitat_common::ui: OutputStream(stdout): { is_colored(): true, supports_color(): true, is_a_terminal(): true }
DEBUG:habitat_common::ui: OutputStream(stderr): { is_colored(): true, supports_color(): true, is_a_terminal(): true }
DEBUG:habitat_core::os::process::imp: Calling execvp(): ("/hab/pkgs/core/hab-sup/0.16.0/20170119213934/bin/hab-sup") ["config", "core/logstash"]
DEBUG:hab_sup: clap matches ArgMatches { args: {}, subcommand: Some(SubCommand { name: "config", matches: ArgMatches { args: {"pkg_ident_or_artifact": MatchedArg { occurs: 1, vals: {1: "core/logstash"} }}, subcommand: None, usage: Some("USAGE:\n    hab-sup config [FLAGS] <pkg_ident_or_artifact>") } }), usage: Some("USAGE:\n    hab-sup [FLAGS] [SUBCOMMAND]") }
DEBUG:hab_sup: subcommand name "config"
DEBUG:hab_sup: Subcommand matches ArgMatches { args: {"pkg_ident_or_artifact": MatchedArg { occurs: 1, vals: {1: "core/logstash"} }}, subcommand: None, usage: Some("USAGE:\n    hab-sup config [FLAGS] <pkg_ident_or_artifact>") }
DEBUG:hab_sup: Config:
Config { http_listen_addr: ListenAddr(V4(0.0.0.0:9631)), gossip_listen: GossipListenAddr(V4(0.0.0.0:9638)), command: Config, package: PackageIdent { origin: "core", name: "logstash", version: None, release: None }, local_artifact: None, url: "https://willem.habitat.sh/v1/depot", topology: Standalone, group: "default", path: "/hab/svc/hab-sup/data", archive: "", bind: [], key: "", email: None, expire_days: None, userkey: None, servicekey: None, infile: None, outfile: None, gossip_peer: [], gossip_permanent: false, update_strategy: None, service_group: "", file_path: "", version_number: 0, organization: None, ring: None, config_from: None }
hab-sup(ER)[src/error.rs:372:8]: No such file or directory (os error 2)		
```

- [ ] Current Habitat environment variables where the `hab` command or supervisor is running.  These can be gathered using:

		env | grep HAB | grep -v HAB_AUTH_TOKEN

(no environment is present - this command returns nothing)

**Comment by eeyun** [comment]
@wdpoorte Thank you for opening this issue!

It is possible that this issue might resolved in  V > 0.16.0. we will test and let you know!

**Comment by smurawski** [comment]
A lot of the configuration loading for the supervisor has changed since 0.16.0 as we move to a multitenant supervisor.  I'll try to replicate, but this may be handled more gracefully now.
